articles/quickstart/native/flutter/interactive.md

```diff
@@ -45,42 +45,42 @@ flutter pub get auth0-flutter
 
 ## Configure Android {{{ data-action=code data-code="build.gradle#11" }}}
 
-If you are not developing for the Android platform, you can skip this step.
+If you are not developing for the Android platform, skip this step.
 
 The SDK requires manifest placeholders. Auth0 uses placeholders internally to define an `intent-filter`, which captures the authentication callback URL. You must set the Auth0 tenant domain and the callback URL scheme.
 
-You do not need to declare a specific `intent-filter` for your activity, because you have defined the manifest placeholders with your Auth0 **Domain** and **Scheme** values and the library will handle the redirection for you.
+You do not need to declare a specific `intent-filter` for your activity because you defined the manifest placeholders with your Auth0 **Domain** and **Scheme** values. The library handles the redirection for you.
 
-In the snippet, we are using values referenced from `strings.xml`:
+The sample uses values from `strings.xml`:
 
-- `com_auth0_domain`: The domain of your Auth0 tenant. Generally, you can find this in the Auth0 Dashboard under your Application's Settings in the Domain field. If you are using a custom domain, you should set this to the value of your custom domain instead.
+- `com_auth0_domain`: The domain of your Auth0 tenant. Generally, you find this in the Auth0 Dashboard under your Application's **Settings** in the Domain field. If you are using a custom domain, you should set this to the value of your custom domain instead.
 - `com_auth0_scheme`: The scheme to use. Can be a custom scheme, or `https` if you want to use [Android App Links](https://auth0.com/docs/applications/enable-android-app-links). You can read more about setting this value in the [Auth0.Android SDK README](https://github.com/auth0/Auth0.Android#a-note-about-app-deep-linking).
 
 Run **Sync Project with Gradle Files** inside Android Studio to apply your changes.
 
 ## Configure iOS
 
-If you are not developing for the iOS platform, you can skip this step.
+If you are not developing for the iOS platform, skip this step.
 
 You need to register your bundle identifier as a custom URL scheme so the callback and logout URLs can reach your app.
 
-In Xcode, go to the **Info** tab of your app target settings. In the **URL Types** section, click the **＋** button to add a new entry. There, enter `auth0` into the **Identifier** field and `$(PRODUCT_BUNDLE_IDENTIFIER)` into the **URL Schemes** field.
+In Xcode, go to the **Info** tab of your app target settings. In the **URL Types** section, select the **＋** button to add a new entry. Then enter `auth0` into the **Identifier** field and `$(PRODUCT_BUNDLE_IDENTIFIER)` into the **URL Schemes** field.
```

Comment:
Is it necessary to change "click" with "select" given that Xcode is a desktop app, so no need ton be mindful of a potential multi-device environment?

Reply:
cc @avanscoy

Reply:
I understand your point here @Widcket. I don't have strong feelings towards this but leaning towards the suggested change more for consistency and that "click" is discouraged in the style guide, albeit for the exact point you're making. I'll leave this one for @avanscoy.

Reply:
Although, is it possible to use a screen assist or something like VoiceOver with Xcode? If that's true then "select" makes sense.

Reply:
Hello @Widcket and @stevehobbsdev ! I suggested "select" to be in-line with the other quickstarts. It would be strange to have one QS with "click" and the rest with "select" so more of a consistency thing.